examples/empirical_frechet_mean_uncertainty_sn.py

```diff
@@ -136,7 +138,8 @@ def plot_modulation_factor(n_samples, dim, n_expectation=1000, n_theta=20):
             n_samples, theta_i, dim, n_expectation=n_expectation)
         measured_modulation_factor.append(var)
         error.append(std_var)
-        print(n_samples, theta_i, var, std_var, '\n')
+        logging.info(
```

Comment:
Is \n needed? Same questions for next `logging.info`

Reply:
For this case I was trying to keep the spacing of the console messages the same way it was spaced before I made my changes as shown below.
![q1](https://user-images.githubusercontent.com/56150991/75583406-5fbf9380-5a22-11ea-94fe-74ba75e1a71e.png)
the \n is not needed and I can remove it if you’d like. If I remove the \n more messages can fit on the screen, so it has that advantage. What do you think?

Reply:
Got it, yes it's easier to read this way. Thanks for clarifying!